common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -251,7 +252,8 @@ public static ILocalAuthenticationResult submitAcquireTokenSilentSync(@NonNull f
             if (BuildConfig.DISABLE_ACQUIRE_TOKEN_SILENT_TIMEOUT){
                 commandResult = submitSilentReturningFuture(command).get();
             } else {
-                commandResult = submitSilentReturningFuture(command).get(ACQUIRE_TOKEN_SILENT_DEFAULT_TIMEOUT_MILLISECONDS, TimeUnit.MILLISECONDS);
+                final int silentTokenTimeOutMs = CommonFlightManager.getIntValue(CommonFlight.ACQUIRE_TOKEN_SILENT_TIMEOUT_MILLISECONDS);
```

Comment:
How are we planning to target teams devices for this flight? Is there any specific version of teams app that we could use?

Reply:
We have a Manufacturer Filter which we can use for request from TeamsDevices.

Reply:
Do we have a list of all manufacturers that are facing this issue? I think Teams devices folks have a big list of all manufacturers.